### ISSUE TITLE
Bug in "Create a combined version of Public Cert and Intermediate and Root CAs"

### ISSUE THREAD

**Issue by balajisivaraman** [sovereign/sovereign#449]
**P.S**: This took me a while to figure out, which is why this issue description is more verbose than is necessary.

I recently faced this issue when I renewed my SSL certificates from AlphaSSL. I copied over the new private and public keys to `roles/common/files` and everything was fine in the web-facing portion (`read`, `blog`, `mail` etc). However, my Dovecot started failing with private key issues. So I started going through the usual phases of debugging whether my private and public keys match, testing my certificate with the `pem` file and so on.

When all those worked as expected (as in no issues with my keys themselves), I turned to the sovereign repository to figure out what could be causing the issue (first by checking reported issues and then by looking at commits), which is when I landed on [this](https://github.com/sovereign/sovereign/commit/ca8a3713207206998aecdbf00894a64e0ec8cf3e) commit.

Long story short, I had to remove the `/etc/ssl/certs/wildcard_combined.pem` file already present in the server so that Ansible would create a new one for me. The issue is because Ansible doesn't check for changed contents in the `shell` command we are running, instead it checks only for presence or absence of the file.

The Ansible [assemble](http://docs.ansible.com/ansible/assemble_module.html) command seems like a possible fix, but I'm not sold on it yet. Any other options to fixing this issue?

**Comment by neuhaus** [comment]
Duplicate of #412 which is a duplicate of #190 - the problem is not fixed so I'm not closing it.

**Comment by neuhaus** [comment]
Resolved by #430
